uyuni/common-libs/common/context_managers.py

```diff
@@ -0,0 +1,29 @@
+"""Collection of context managers for Uyuni."""
+from contextlib import contextmanager
+from spacewalk.common.rhnConfig import CFG, initCFG
+
+@contextmanager
+def cfg_component(component, root=None, filename=None):
+    """Context manager for rhnConfig.
```

Comment:
I think you need to add this file to the "Makefile" to get it into the package and installed.

Reply:
Good catch! I added it to the Makefile now and did a build locally. Now it's in the package :) The spec packages all files under `{pythonXroot}`, no need to change it.